common/browser_context_options_test.go

```diff
@@ -4,13 +4,15 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+
+	"github.com/grafana/xk6-browser/k6"
 )
 
 func TestBrowserContextOptionsPermissions(t *testing.T) {
-	vu := newMockVU(t)
+	vu := k6.NewMockVU(t)
```

Comment:
If I can make it `k6test`, then it makes more sense to name `k6test.NewVU`. Like `httptest.Server` instead of `httptest.MockServer`. The pkg name would already reflect that it's a type for tests.

Reply:
Nice, ❤️ that 🙂